<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>8</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_code">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_code"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_calc">
         <property name="text">
          <string>CALC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item row="6" column="4">
          <widget class="QLabel" name="label_b14">
           <property name="text">
            <string>bit 14</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_b9">
           <property name="text">
            <string>bit 9</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pushButton_b5">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="pushButton_b13">
           <property name="text">
            <string>13</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_b0">
           <property name="text">
            <string>bit 0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="pushButton_b14">
           <property name="text">
            <string>14</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="pushButton_b11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QPushButton" name="pushButton_b15">
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_b2">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_b0">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pushButton_b10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="pushButton_b7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_b4">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_b5">
           <property name="text">
            <string>bit 5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_b1">
           <property name="text">
            <string>bit 1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="label_b13">
           <property name="text">
            <string>bit 13</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="pushButton_b12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_b8">
           <property name="text">
            <string>bit 8</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="b7">
           <property name="text">
            <string>bit 7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_b2">
           <property name="text">
            <string>bit 2</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_b6">
           <property name="text">
            <string>bit 6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_b10">
           <property name="text">
            <string>bit 10</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButton_b6">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButton_b9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_b3">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_b4">
           <property name="text">
            <string>bit 4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_b3">
           <property name="text">
            <string>bit 3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_b8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_b1">
           <property name="baseSize">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_b12">
           <property name="text">
            <string>bit 12</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_b15">
           <property name="text">
            <string>bit 15</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_b11">
           <property name="text">
            <string>bit 11</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
